addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -1439,6 +1439,51 @@ export function makeActionManager(env, router = _router) {
             }
             throw new InvalidButtonParamsError("Missing type for doActionButton request");
         }
+        if (action.embedded_action_ids && !this.currentController?.config.currentEmbeddedActionId) {
+            const embeddedActionsOrder = JSON.parse(
+                browser.localStorage.getItem(
+                    `orderEmbedded${action.id}+${params.resId || ""}+${user.userId}`
+                )
+            );
+            const embeddedActionId = embeddedActionsOrder?.at(0);
+            const embeddedAction = action.embedded_action_ids?.find(
+                (embeddedAction) => embeddedAction.id === embeddedActionId
+            );
+            if (embeddedAction) {
+                const embeddedActions = [
+                    ...action.embedded_action_ids,
+                    {
+                        id: false,
+                        name: action.name,
+                        parent_action_id: action.id,
+                        parent_res_model: action.res_model,
+                        action_id: action.id,
+                        user_id: false,
+                        context: {},
```

Comment:
shouldn't this be action.context?

Reply:
I am not sure to be fair. I tried to imitate what's going on [here](https://github.com/odoo/odoo/blob/master/addons/web/static/src/search/control_panel/control_panel.js#L83).